Makefile

```diff
@@ -1,9 +1,9 @@
+SHELL = bash
+
 # Current Operator version
 VERSION ?= 0.0.1
 # Default bundle image tag
 BUNDLE_IMG ?= controller-bundle:$(VERSION)
-# Default platform for bundle
-PLATFORM="kubernetes"
```

Comment:
This variable is still used in target bundle.

Reply:
Re-added